Event/NotificationEventInterface.php

```diff
@@ -6,12 +6,12 @@
 
 interface NotificationEventInterface
 {
-    const NOTIFICATION_CREATED = 'kopaygorodsky.notification.message.created';
-    const NOTIFICATION_POST_PERSIST = 'kopaygorodsky.notification.message.postPersist';
-    const JOB_CREATED = 'kopaygorodsky.notification.job.created';
-    const JOB_PRE_SEND = 'kopaygorodsky.notification.job.preSend';
-    const JOB_POST_SEND = 'kopaygorodsky.notification.job.postSend';
-    const JOB_FAILED = 'kopaygorodsky.notification.job.failed';
+    const NOTIFICATION_CREATED = 'kopay.notify.message.created';
+    const NOTIFICATION_POST_PERSIST = 'kopay.notify.message.postPersist';
```

Comment:
symfony convention is snake case for event names, example - https://github.com/symfony/form/blob/master/FormEvents.php#L34

Reply:
it's original naming from author, but I can fix that, no prob.